src/applications/personalization/profile-2/components/ProfileHeader.jsx

```diff
@@ -113,14 +108,13 @@ const ProfileHeader = ({
             <img
               src={SERVICE_BADGE_IMAGE_PATHS.get(latestBranchOfService)}
               alt={`${latestBranchOfService} seal`}
-              className="profile-service-badge"
+              className="profile-service-badge vads-u-padding-right--3"
               aria-hidden="true"
               role="presentation"
             />
           )}
         </div>
         <div className="name-and-title-wrapper">
-          <h1 className={classes.title}>Your Profile</h1>
```

Comment:
😱 no `h1` on the page anymore! Do we have a ticket to redo the sidenav or are we waiting on Tressa to finish up mobile nav design changes first?

Reply:
Just messaged Tressa to confirm, but I will wait with merging this ticket till this is sorted out